### ISSUE TITLE
Recursive Bisect

### ISSUE THREAD

**Issue by ryneeverett** [ryneeverett/git-regress#4]

**Comment by ryneeverett** [comment]
This will require more thorough testing with more commits on the example repo. Postpone until after #9.

**Comment by ryneeverett** [comment]
This will require more thorough testing with more commits on the example repo. Postpone until after #15.

**Comment by ryneeverett** [comment]
This will require more thorough testing with more commits on the example repo. Postpone until after #22 .

**Comment by ryneeverett** [comment]
I'm not sure this would really be such a useful feature. If the result of a single bisect isn't good enough, then bisect probably isn't the right strategy.
